jasmine-2.5.2/spec/ScoreSpec.js

```diff
@@ -0,0 +1,34 @@
+describe("Score", function() {
+  var score;
+
+  beforeEach(function() {
+    score = new Score();
+  });
+  it('initialises default settings', function() {
+    expect(score.getHits()).toEqual(0);
+    expect(score.getBonus()).toEqual(0);
+    expect(score.getRollTotal()).toEqual(0);
+    expect(score.getRunningTotal()).toEqual(0)
+  });
+  it('records current hits and bonus', function() {
+    score.setHits(4);
+    score.setBonus(2);
```

Comment:
This is a very nice way of handling strikes and spares. Still, there's some naming hiding behind the number.